Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/SiteSettingsController.cs

```diff
@@ -394,10 +398,11 @@ public HttpResponseMessage UpdateDefaultPagesSettings(UpdateDefaultPagesSettings
                 portalInfo.TermsTabId = this.ValidateTabId(request.TermsTabId, pid);
                 portalInfo.PrivacyTabId = this.ValidateTabId(request.PrivacyTabId, pid);
                 PortalController.Instance.UpdatePortalInfo(portalInfo);
-
-                PortalController.UpdatePortalSetting(pid, "Redirect_AfterLogin", this.ValidateTabId(request.RedirectAfterLoginTabId, pid).ToString(), false, cultureCode);
-                PortalController.UpdatePortalSetting(pid, "Redirect_AfterLogout", this.ValidateTabId(request.RedirectAfterLogoutTabId, pid).ToString(), false, cultureCode);
-                PortalController.UpdatePortalSetting(pid, "Redirect_AfterRegistration", this.ValidateTabId(request.RedirectAfterRegistrationTabId, pid).ToString(), false, cultureCode);
+
+                var portalSettings = PortalController.Instance.GetCurrentSettings();

```

Comment:
I think maybe this line was added by mistake @valadas - can you confirm?

Reply:
Good catch, fixed